Classes/Domain/Site/Site.php

```diff
@@ -150,33 +142,24 @@ public function getDefaultLanguage()
     }
 
     /**
-     * Generates a list of page IDs in this site. Attention, this includes
-     * all page types! Deleted pages are not included.
-     *
-     * @param int|string $rootPageId Page ID from where to start collection sub pages
-     * @param int $maxDepth Maximum depth to descend into the site tree
-     * @return array Array of pages (IDs) in this site
+     * @inheritDoc
      */
-    public function getPages($rootPageId = 'SITE_ROOT', $maxDepth = 999)
+    public function getPages(
+        ?int $pageId = null,
```

Comment:
I think this change could be breaking, as getPages() is a public method and so far accepted int and string values. 

String values, especially the keyword SITE_ROOT could be used in third-party extensions and would stop working with these changes, what do you think?

Reply:
That PR is for master aka 11.1.x, so the BC should be fine for that.
The origianal PR #2898 should go onto release-11.0.x instead of that PR.

Reply:
Ok, fine for me as there are breaking changes anyway

But we should mention this in the release notes, please as add discused